GameOfLifeApp/Controllers/GameLogic.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+
+namespace GameOfLifeApp 
+{
+    /// <summary>
+    /// Generates fields.
+    /// </summary>
+    public class GameLogic : IGameLogic
+    {
+        /// <summary>
+        /// Generates a start array of random values according to user`s input.
+        /// </summary>
+        public void SetUpField(IGameField field)
+        {                
+            Random rnd = new Random();
+
+            bool[,] InitialField = new bool[field.CountOfRows, field.CountOfColumns];
+
+            for (int row = 0; row < field.CountOfRows; row++)
+            {
+                for (int column = 0; column < field.CountOfColumns; column++)
+                {
+                    InitialField[row, column] = rnd.Next(0, 2) == 0;
+                }
+            }
+            field.CurrentField = InitialField;
+        }
+
+        /// <summary>
+        /// Populates the field with Glider the Shape. 
+        /// </summary>
+        public void CreateGliderShape(IGameField pr)
+        {
+            pr.CurrentField[0, 0] = true;
```

Comment:
Don't shorten variable names.
It's very confusing can't understand what's pr in this context.

Reply:
pr -> field